README.md

```diff
@@ -28,6 +28,24 @@ Then copy the `./bin/swctl-latest-(darwin|linux|windows)-amd64` to your `PATH` d
 usually `/usr/bin/` or `/usr/local/bin`, or you can copy it to any directory you like,
 and add that directory to `PATH`, we recommend you to rename the `swctl-latest-(darwin|linux|windows)-amd64` to `swctl`.
 
+In addition, you can also quickly install through the following methods:
+
+**Install swctl on Linux or macOS**
+
+```shell
+/bin/bash -c "$(curl -fsSL https://github.com/apache/skywalking-cli/tree/master/scripts/install.sh)"
```

Comment:
This is not a valid URL https://github.com/apache/skywalking-cli/tree/master/scripts/install.sh

Reply:
Because it is not merged yet, and should be https://raw.githubusercontent.com/apache/skywalking-cli/tree/master/scripts/install.sh